scripts/smoke.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import yyjson
+
+inv = b'["bar\x80"]'.decode('utf8', errors='replace')
```

Comment:
This unicode support should become part of the unit tests suite so we can ensure that it will work across different python versions / compiler options.